.github/workflows/build.yaml

```
@@ -423,7 +424,7 @@ jobs:
     if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
     needs:
       - build-base
-    timeout-minutes: 30
+    timeout-minutes: 60
```

```suggestion
    timeout-minutes: 60
```

Reason for this? Publishing  the helm charts should be ~5 minutes at most, unless we're doing something wrong.

---

Oh this isn't the chart pub step, nevermind.